index.js

```diff
@@ -41,7 +47,15 @@ function getAllMovieTitles() {}
  *  getHighestMetascore(movies);
  *  //> 96
  */
-function getHighestMetascore() {}
+function getHighestMetascore(movies) {
+  let highestMetascore = 0
+  for (let i = 0; i < movies.length; i++) {
```

Comment:
I might use a `for...of` loop here since we aren't actually using `i` for anything except accessing elements in our array.

Reply:
Got it! and thank you for the feedback